scripts/app.js

```diff
@@ -0,0 +1,93 @@
+const StagingUtils = require('./stagingUtils');
+
+async function main() {
+  const patchFlag = process.argv[2];
+  const buildSize = process.argv[3];
+
+  this.validateConfiguration();
```

Comment:
this line is breaking as the validateConfiguration method isn't local to the file anymore.